### PR TITLE
Updated the navigation UI and theme

### PR DESCRIPTION
**Pull request by BidemiEnoch**

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -17,7 +17,8 @@
         "react-scripts": "^4.0.3",
         "redux": "^4.0.5",
         "redux-thunk": "^2.3.0",
-        "styled-components": "^5.2.1"
+        "styled-components": "^5.2.1",
+        "yup": "^0.32.9"
```
Comment (Tekipeps) [inline client/package.json:21]: dont you think formikjs will be better than yup?

**Review by Tekipeps — COMMENTED** [review]
Chekout [https://formik.org/](formik) and let me know if we should go ahead with yup

File: client/package.json (hunk 1 above)
Comment (BidemiEnoch) [reply client/package.json:21]: Ive used formik b4, it has a plugin for yup called validationSchema or something... and yes, it makes displaying error messages and fetching input values easier. Lets use yup for now, if the in the future, we feel the form has become quite complex, formik can be introduced to make our lives easier, for now... I don't see a need anyways.

Comment (BidemiEnoch) [reply client/package.json:21]: with formik you still have to manually define your own validation functions which means we would most likely still use yup with formik so...

**Pull request merged**